front-end/src/shared/GlobalStyle.js

```diff
@@ -0,0 +1,19 @@
+import { createGlobalStyle } from 'styled-components';
```

Comment:
If you don't mind talking about this at the team meeting, I would be interested in learning more about the benefits of styled-components vs. css.

Reply:
Will do for sure!